crates/solidity/outputs/npm/package/src/generated/index.d.ts

```diff
@@ -10,5290 +10,612 @@
-    spawn(): Cursor;
-    get isCompleted(): boolean;
-    get node(): RuleNode | TokenNode;
-    get textOffset(): TextIndex;
-    get textRange(): TextRange;
-    get pathRuleNodes(): Array<RuleNode>;
-    goToNext(): boolean;
-    goToNextNonDescendent(): boolean;
-    goToPrevious(): boolean;
-    goToParent(): boolean;
-    goToFirstChild(): boolean;
-    goToLastChild(): boolean;
-    goToNthChild(childNumber: number): boolean;
-    goToNextSibling(): boolean;
-    goToPreviousSibling(): boolean;
-    findTokenWithKind(kinds: Array<TokenKind>): TokenNode | null;
-    findRuleWithKind(kinds: Array<RuleKind>): RuleNode | null;
-  }
-  export class Language {
-    constructor(version: string);
-    get version(): string;
-    static supportedVersions(): Array<string>;
-    parse(productionKind: ProductionKind, input: string): ParseOutput;
-  }
-  export class ParseOutput {
-    get parseTree(): RuleNode | TokenNode | null;
-    get errors(): Array<ParseError>;
-    get isValid(): boolean;
-  }
-  export class ParseError {
-    get textRange(): TextRange;
-    toErrorReport(sourceId: string, source: string, withColour: boolean): string;
-  }
-}
-export namespace syntax$nodes {
+export namespace kinds {
   export enum ProductionKind {
-    /**
-     * ## Unversioned
-     *
-     * ```ebnf
-     * ABICoderPragma = ABICODER_KEYWORD IDENTIFIER;
-     * ```
-     */
     ABICoderPragma = "ABICoderPragma",
-    /**
-     * ## Unversioned
-     *
-     * ```ebnf
-     * ABICODER_KEYWORD = "abicoder";
-     * ```
-     */
-    AbicoderKeyword = "AbicoderKeyword",
-    /**
-     * ## v0.6.0
-     *
-     * ```ebnf
-     * ABSTRACT_KEYWORD = "abstract";
-     * ```
-     */
-    AbstractKeyword = "AbstractKeyword",
-    /**
-     * ## Unversioned
-     *
-     * ```ebnf
-     * ADDRESS_KEYWORD = "address";
-     * ```
-     */
-    AddressKeyword = "AddressKeyword",
-    /**
-     * ## Unversioned
-     *
-     * ```ebnf
-     * AddressType = (ADDRESS_KEYWORD PAYABLE_KEYWORD?) | PAYABLE_KEYWORD;
-     * ```
-     */
+    AddSubOperator = "AddSubOperator",
     AddressType = "AddressType",
-    /**
-     * ## Unversioned
-     *
-     * ```ebnf
-     * AMPERSAND = "&";
-     * ```
-     */
-    Ampersand = "Ampersand",
-    /**
-     * ## Unversioned
-     *
-     * ```ebnf
-     * AMPERSAND_AMPERSAND = "&&";
-     * ```
-     */
-    AmpersandAmpersand = "AmpersandAmpersand",
-    /**
-     * ## Unversioned
-     *
-     * ```ebnf
-     * AMPERSAND_EQUAL = "&=";
-     * ```
-     */
-    AmpersandEqual = "AmpersandEqual",
-    /**
-     * ## Unversioned
-     *
-     * ```ebnf
-     * ANONYMOUS_KEYWORD = "anonymous";
-     * ```
-     */
-    AnonymousKeyword = "AnonymousKeyword",
-    /**
-     * ## Unversioned
-     *
-     * ```ebnf
-     * ArgumentsDeclaration = OPEN_PAREN (PositionalArgumentsList | NamedArgumentsDeclaration)? CLOSE_PAREN;
-     * ```
-     */
+    AndOperator = "AndOperator",
     ArgumentsDeclaration = "ArgumentsDeclaration",
```

Comment:
EBNF documentation is now broken/removed from the template. Are they generated beforehand?

Reply:
No, I didn't port the ebnf crate. I was going to do that when I did the spec, because the spec needs the old ebnf crate.